nav2_bringup/params/nav2_multirobot_params_1.yaml

```diff
@@ -45,6 +45,7 @@ bt_navigator:
     odom_topic: /odom
     bt_loop_duration: 10
     default_server_timeout: 20
+    bt_wait_for_service_timeout: 1000.0
```

Comment:
Make sure to remark that the units are ms in the config guide

Reply:
@embeddedadam your config guide PR has this an an `int`, but this is clearly a `double`. This will cause a mismatched type exception to be thrown. Remove the `.0` from the config files

Reply:
@SteveMacenski yes I am aware of that, I fixed everything in new patch except uts.

Reply:
OK! Please push that so I can take a look :-)